.github/workflows/update-artifacts.yml

```diff
@@ -73,31 +73,31 @@ jobs:
         run: |
           pip install -r scripts/upload-artifacts/requirements.txt
           pip install -r scripts/generate-artifacts-manifest/requirements.txt
-          
-      - name: Check if truefoundry chart changed
-        id: check_truefoundry_chart_update
-        run: |
-          if git diff --name-only origin/main...HEAD | grep -q '^charts/truefoundry/Chart.yaml$'; then
-            echo "changed=true" >> $GITHUB_OUTPUT
-          else
-            echo "changed=false" >> $GITHUB_OUTPUT
-          fi
 
-      - name: Copy the current artifacts manifest file to old-artifacts.json for changelog
-        if: steps.check_truefoundry_chart_update.outputs.changed == 'true'
+
+      - name: Copy current manifest for changelog
+        if: always()
         run: |
           cp charts/tfy-k8s-aws-eks-inframold/artifacts-manifest.json charts/tfy-k8s-aws-eks-inframold/old-artifacts-manifest.json
 
+
       # Generate artifacts manifest for inframold charts
       - name: Generate Artifacts Manifest for Each Chart
         run: |
-          charts_list=("tfy-k8s-aws-eks-inframold" "tfy-k8s-azure-aks-inframold" "tfy-k8s-gcp-gke-standard-inframold" "tfy-k8s-civo-talos-inframold" "tfy-k8s-generic-inframold")
-          for chart in "${charts_list[@]}";
-          do
+          charts_list=(
+            "tfy-k8s-aws-eks-inframold"
+            "tfy-k8s-azure-aks-inframold"
+            "tfy-k8s-gcp-gke-standard-inframold"
+            "tfy-k8s-civo-talos-inframold"
+            "tfy-k8s-generic-inframold"
+          )
+          for chart in "${charts_list[@]}"; do
             version=$(cat charts/$chart/Chart.yaml | grep version | awk '{print $2}')
-            
-            python scripts/generate-artifacts-manifest/artifacts_template_generator.py $chart https://truefoundry.github.io/infra-charts/ \
-              $version charts/$chart/values-artifact-manifest.yaml charts/$chart/artifacts-manifest.json scripts/generate-artifacts-manifest/extra.json
+          
+            python scripts/generate-artifacts-manifest/artifacts_template_generator.py \
+              $chart https://truefoundry.github.io/infra-charts/ \
+              $version charts/$chart/values-artifact-manifest.yaml charts/$chart/artifacts-manifest.json \
+              scripts/generate-artifacts-manifest/extra.json
```

Comment:
These changes are not required - can you please remove these formatting. It add unnecessary lines to check